src/PrestaShopBundle/Form/Admin/Product/ProductShipping.php

```diff
@@ -150,6 +150,7 @@ public function buildForm(FormBuilderInterface $builder, array $options)
                 'weight',
                 FormType\NumberType::class,
                 [
+                    'scale' => 6,
```

Comment:
the fix LGTM, but I think it would be nice to have this value stored in a constant that could be reused for weight precision, instead of potentially hardcoding it at several places, what do you think ?

Reply:
hmmm, you're right, what about place, where you think would be best to put it? `CommonAbstractType`?

Reply:
As it's relative to this form, put it in the form.

Reply:
Agreed with peers 😄  a constant inside ProductShipping would be great

Reply:
Sorry guys but I added it to `CommonAbstractType` as this const must be used also in `ProductCombination`, I've decided to follow constant added for price precision

Let me know if that's ok or we should have dedicated constant in both forms separately for some reason